web/src/toga_web/widgets/numberinput.py

```diff
@@ -0,0 +1,50 @@
+from .base import Widget
+
+
+class NumberInput(Widget):
+    def create(self):
+        self._return_listener = None
+        self.native = self._create_native_widget("sl-input")
+        self.native.type = "number"
+        self.native.value = None
+        self.native.onblur = self.lost_focus
```

Comment:
We have a convention of clearly naming native event handlers based on their *native* naming - so in this case, `lost_focus()` should be `dom_onblur()`. That makes it easier for us to follow when/why certain methods are being invoked.